src/Service/TimestreamQuery/composer.json

```diff
@@ -14,7 +14,8 @@
         "php": "^7.2.5 || ^8.0",
         "ext-filter": "*",
         "ext-json": "*",
-        "async-aws/core": "^1.9"
+        "async-aws/core": "^1.9",
+        "symfony/polyfill-uuid": "^1.0"
```

Comment:
Good catch. I've automated code generation for this.

Reply:
👌